nextcord/application_command.py

```diff
@@ -1265,77 +1308,173 @@ def __init__(
             cmd_arg_given = False
 
         self.name = cmd_arg.name or parameter.name
+        # Use the given name, or default to the parameter name.
+        # typehint_origin = typing.get_origin(parameter.annotation)  # TODO: Once Python 3.10 is standard, use this.
+        typehint_origin = typing_extensions.get_origin(parameter.annotation)
+
+        annotation_type: ApplicationCommandOptionType = MISSING
```

Comment:
This value is never used
```suggestion
        annotation_type: ApplicationCommandOptionType
```